clang/test/SemaSYCL/keyword.cpp

```diff
@@ -0,0 +1,6 @@
+// RUN: %clang_cc1 -verify -fsyntax-only -std=c++11 %s
```

Comment:
```suggestion
// RUN: %clang_cc1 -verify -fsyntax-only %s
```
`-std=c++11` option is not supported on Windows.

Reply:
-std=c++11 is still a cc1 option though, so it's completely valid in a test.

Reply:
@erichkeane, you are right.
I was confused by SYCL tests, which include C++ standard library headers, which can't be compiled with this option.
This test should work with option on Windows platform too.